src/pyeantic/pyeantic/cppyy_eantic.py

```diff
@@ -63,6 +63,7 @@ def inplace(lhs, *args, **kwargs): raise NotImplementedError("inplace operators
             setattr(proxy, "__r%s__"%n, binary)
             setattr(proxy, "__i%s__"%n, inplace)
         setattr(proxy, "__neg__", lambda self: cppyy.gbl.eantic.minus(self))
+        setattr(proxy, "__pow__", lambda self, n: cppyy.gbl.eantic.pow(self, int(n)))
```

Comment:
Are you sure about the int cast here? Shouldn't we just fail if it's not an int? I am not sure that `a**"1"` should work.

Reply:
True. The case I had in mind was #9. Let me remove that from this pull request.